docs/features/publish-to-github.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: "Publish to GitHub"
+description: "Learn more about the Publish to GitHub feature"
+---
+
+This integration allows users to publish their projects from the CodeCrafters platform to GitHub with just a couple of clicks.
+
+Each course available on CodeCrafters includes an option in the language dropdown menu to **"Publish to GitHub"**:
+
+<Frame>
+  <img src="../img/features/publish-to-github-dropdown.png"></img>
+</Frame>
+
+## Frequently Asked Questions
+
+### Is there a limit to how many projects I can publish to GitHub?
+
+No, you can publish as many projects as you like.
+
+## Who can see my published projects?
+
+This is up to you. You can set this in the privacy settings of your GitHub repository.
+
+### Can I update a project once it's been published?
+
+Yes! Any subsequent changes made to your Codecrafters project will be automatically synced with your GitHub repo.
```

Comment:
Changes can be made via the CLI and via Git push - maybe there's a way to be more accurate here and say that we'll keep the `master` branch in-sync?